README.md

```diff
@@ -412,6 +412,52 @@ You can also [join our Discord channel #development](https://discord.gg/JuhHdHTt
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
+### Submitting issues
+
+You can submit issue to the project but you should do it in a way that helps developers to resolve it as quickly as possible.
+
+For that, you need to add as much valuable informations as possible.
+
+You can have this valuable informations by doing this steps :
+
+- Go to the root of the git cloned project
+- Edit `web/entrypoint.sh` and add at the top `export DEBUG=1`
+This should give you this result
+  ```python
+  #!/bin/bash
+
+  export DEBUG=1
+
+  python3 manage.py migrate
+  python3 manage.py runserver 0.0.0.0:8000
+
+  exec "$@"
+  ```
+- Restart the web container `docker-compose restart web`
+- To deactivate set **DEBUG** to **0** and restart web container again
+
+Then, with **DEBUG** set to **1**, in the `make logs` output you could see the full stack trace to debug reNgine.
+
+Example with the tool arsenal version check API bug.
+```
+web_1          |   File "/usr/local/lib/python3.10/dist-packages/celery/app/task.py", line 411, in __call__
```

Comment:
```suggestion
Example with the tool arsenal version check API bug.

```
web_1          |   File "/usr/local/lib/python3.10/dist-packages/celery/app/task.py", line 411, in __call__
```